.travis.yml

```diff
@@ -20,14 +20,16 @@ env:
 os:
   - linux
 rust:
-  - nightly
+  # we force the Rust version to nightly-2017-12-20 because of a Ring package issue that broke Rocket on the latest Rust version (https://github.com/briansmith/ring/issues/598);
+  # TODO: #32 the used version must be set back to nightly as soon as the fix is available
+  - nightly-2017-12-20
```

Comment:
as we compile directly from docker , I don't thikn it's important to put it here, as we don't use the rustc from the host